docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '3.1'
```

Comment:
This is because 3.8 did not work for me for some reason :(

Reply:
yes , probably you need to upgrade your docker engine version. Check this link https://docs.docker.com/compose/compose-file/compose-versioning/